### DISCUSSION TITLE
GitHub Education Communicate using Markdown task i cant edit the file index.md

### DISCUSSION THREAD

**Discussion by oussema-benelhadj** [community/community#129442] [GitHub Education]
### Hi everyone,

Hello
I can not edit the index.md file only if i am in a branch how to enter a branch.
Could someone explain me how or a vedio tuto

**Comment by Carlos-Levi** [comment]
Hi,  There are two main approaches to enter a branch:

1. Using the GitHub website:

a. Navigate to the desired repository on GitHub.com.
b. In the file tree view on the left side, locate the branch dropdown menu.
c. You might see the current branch displayed there. Click on it or the dropdown arrow.
d. If there are existing branches, you'll see a list. Choose the branch you want to work on (where you want to edit index.md).
e. If you need to create a new branch, click on "New branch" and follow the prompts to name it and choose a source branch (usually master).

2. Using Git commands (assuming they have Git installed):

a. Open a terminal or command prompt.
b. Navigate to the local directory containing the cloned repository using the cd command.
c. Use the git branch command to see a list of available branches.
d. To switch to an existing branch, use the git checkout <branch_name> command, replacing <branch_name> with the actual branch name.
e. To create a new branch and switch to it at the same time, use git checkout -b <new_branch_name>.

I Recommend the GitHub website approach for beginners, as it's more user-friendly.

https://m.youtube.com/watch?v=e2IbNHi4uCI
